appengine/README.txt

```diff
@@ -35,7 +35,7 @@ blockly/
      `- python_compressed.js
 
 Instructions for fetching the optional Closure library may be found here:
-  https://developers.google.com/blockly/hacking/closure
+  https://developers.google.com/blockly/guides/modify-blockly/web/closure
```

Comment:
Why is the new URL 'modify-blockly' and 'configure-blockly'?  These are the Blockly docs, the extra 'blockly' seems awkwardly redundant.